tests/test_neighborhood.py

```diff
@@ -12,18 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from unittest.mock import MagicMock
+
 from model_analyzer.config.generate.neighborhood import Neighborhood
 from model_analyzer.config.generate.search_config import NeighborhoodConfig
 from model_analyzer.config.generate.search_dimension import SearchDimension
 from model_analyzer.config.generate.search_dimensions import SearchDimensions
 from model_analyzer.config.generate.coordinate_data import CoordinateData
 from model_analyzer.config.generate.coordinate import Coordinate
 
+from .common.test_utils import construct_run_config_measurement
 from .common import test_result_collector as trc
 
 
 class TestNeighborhood(trc.TestResultCollector):
 
+    def _construct_rcm(self, throughput: float, latency: float):
```

Comment:
2 files with the same function. Let's move it to utils.

Reply:
I think other tests would benefit from a slimmed down method like this, but we can leave it as is for now.